xbmc/guilib/GUIBaseContainer.h

```diff
@@ -152,8 +154,8 @@ class CGUIBaseContainer : public IGUIContainer
 
   int m_pageControl;
 
-  std::vector<CGUIListItemLayout> m_layouts;
-  std::vector<CGUIListItemLayout> m_focusedLayouts;
+  std::list<CGUIListItemLayout> m_layouts;
+  std::list<CGUIListItemLayout> m_focusedLayouts;
```

Comment:
Bullshit! A typical cache line size is 64 bytes. CGUIListItemLayout's size is 560 bytes. Beyond one such instance, there is no locality.